DscResources/MSFT_Archive/MSFT_Archive.psm1

```diff
@@ -85,26 +91,26 @@ function Get-TargetResource
     [CmdletBinding()]
     param
     (
-        [Parameter(Mandatory = $true)]
+        [Parameter(Mandatory = $true, HelpMessage="The path to the archive file that should be expanded to or removed from the specified destination.")]
```

Comment:
Most other resources only have HelpMessages added only to Set, but these are added in Get?